ops/services/web_application_firewall/main.tf

```diff
@@ -78,10 +86,33 @@ resource "azurerm_web_application_firewall_policy" "sr_waf_policy" {
       selector_match_operator = "Equals"
     }
 
+    exclusion {
+      match_variable          = "RequestArgNames"
+      selector                = "iss"
+      selector_match_operator = "Equals"
+    }
+    exclusion {
+      match_variable          = "RequestArgNames"
+      selector                = "street" //Will need to turn on 942440 and 942110 if not effective.
+      selector_match_operator = "Equals"
+    }
+    exclusion {
+      match_variable          = "RequestArgNames"
+      selector                = "namePrefixMatch" //Will need to turn on 942330 if not effective
```

Comment:
is this comment referring to turning on the override as in turning off the rule?

Reply:
That's correct! It would need to be added to the exclusion list lower in this file. My apologies for the confusing comment...this was more for my own reference as I work to narrow down these false positives!